doc/source/how-to/documenting.rst

```diff
@@ -478,16 +486,40 @@ The resulting PDF and intermediate LaTeX files are created in the
 Enable multi-version documentation
```

Comment:
@Revathyvenugopal162 I redid this to make the documentation "timeless" as the original content was now rather dated. Please review.